docs/user_manual/source/integration.rst

```diff
@@ -95,7 +95,8 @@ Parameters
 ----------
 
 .. note::
-   The non-default (i.e. non-zero) settings of ``FPU`` have not been verified yet.
+   All eXtension interface parameters (``X_EXT``, ``X_NUM_RS``, ``X_ID_WIDTH``, ``X_MEM_WIDTH``, ``X_RFR_WIDTH``, ``X_RFW_WIDTH`` and ``X_MISA``)
```

Comment:
The eXtension interface parameters ``X_NUM_RS``, ``X_ID_WIDTH``, ``X_MEM_WIDTH``, ``X_RFR_WIDTH`` and ``X_RFW_WIDTH``